packages/grafana-schema/src/raw/dashboard/x/dashboard_types.gen.ts

```diff
@@ -625,6 +625,36 @@ export interface DataTransformerConfig {
   options: unknown;
 }
 
+/**
+ * Time picker configuration
+ * It defines the default config for the time picker and the refresh picker for the specific dashboard.
+ */
+export interface TimePickerConfig {
+  /**
+   * Whether timepicker is collapsed or not. Has no effect on provisioned dashboard.
+   */
+  collapse: boolean;
```

Comment:
I don't think we have such a persisted state

Reply:
I think we can start with just 
* remove it from cue
* ignoring it in the save model => scene => save model transforms 

So then when you save scene based dashboards they will automatically be removed